conferencias4/src/screens/Add.js

```diff
@@ -8,7 +8,7 @@ function Add({navigation}){
    
     return(
       <View >
```

Comment:
Our mentors recommend me to use `SafeAreaView` component in all the screens components
[https://reactnative.dev/docs/safeareaview](url)